packaging/ansible-runner-service-project/project/roles/ovirt-host-deploy-vdsm/tasks/configure.yml

```diff
@@ -11,6 +11,35 @@
 # grab local devices that should be blacklisted. This may expose
 # the local devices as FC storage in engine.
 
+- block:
+  - name: mask libvirt modular daemon units
+    systemd:
+      name: "{{ item }}"
+      state: stopped
+      enabled: no
+      masked: yes
+    with_items:
+      - virtqemud.socket
+      - virtinterfaced.socket
+      - virtnetworkd.socket
+      - virtnodedevd.socket
+      - virtnwfilterd.socket
+      - virtsecretd.socket
+      - virtstoraged.socket
+      - virtproxyd.socket
+
+  - name: enable libvirtd monolithic daemon unit
+    systemd:
+      name: libvirtd
+      state: started
```

Comment:
We don't want to actually start libvirtd here - just enable it. VDSM starts libvirtd unit using socket activation and libvirt's socket units must start before the daemon itself.

I would try not rebooting hosts once you get rid of `state: started`

Reply:
tried that but still wont work without restarting the host.
get something like:

```
File "/usr/lib/python3.9/site-packages/vdsm/tool/nwfilter.p>
Feb 21 14:34:02 ost-basic-suite-master-host-1 vdsmd_init_common.sh[9134]:     conn = libvirtconnection.get(None, False)
Feb 21 14:34:02 ost-basic-suite-master-host-1 vdsmd_init_common.sh[9134]:   File "/usr/lib/python3.9/site-packages/vdsm/common/libvirtc>
Feb 21 14:34:02 ost-basic-suite-master-host-1 vdsmd_init_common.sh[9134]:     conn = open_connection('qemu:///system', SASL_USERNAME, p>
Feb 21 14:34:02 ost-basic-suite-master-host-1 vdsmd_init_common.sh[9134]:   File "/usr/lib/python3.9/site-packages/vdsm/common/libvirtc>
Feb 21 14:34:02 ost-basic-suite-master-host-1 vdsmd_init_common.sh[9134]:     return function.retry(libvirtOpen, timeout=10, sleep=0.2)
Feb 21 14:34:02 ost-basic-suite-master-host-1 vdsmd_init_common.sh[9134]:   File "/usr/lib/python3.9/site-packages/vdsm/common/function>
Feb 21 14:34:02 ost-basic-suite-master-host-1 vdsmd_init_common.sh[9134]:     return func()
Feb 21 14:34:02 ost-basic-suite-master-host-1 vdsmd_init_common.sh[9134]:   File "/usr/lib64/python3.9/site-packages/libvirt.py", line >
Feb 21 14:34:02 ost-basic-suite-master-host-1 vdsmd_init_common.sh[9134]:     raise libvirtError('virConnectOpenAuth() failed')
Feb 21 14:34:02 ost-basic-suite-master-host-1 vdsmd_init_common.sh[9134]: libvirt.libvirtError: Failed to connect socket to '/var/run/l>
Feb 21 14:34:02 ost-basic-suite-master-host-1 vdsmd_init_common.sh[9095]: vdsm: stopped during execute nwfilter task (task returned wit>
Feb 21 14:34:02 ost-basic-suite-master-host-1 systemd[1]: vdsmd.service: Control process exited, code=exited, status=1/FAILURE

```